packages/react-native/tsconfig.build.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "../../tsconfig.build.json",
+
+  "compilerOptions": {
+    "outDir": "./build",
+    // FIXME https://github.com/hyperledger/aries-framework-javascript/pull/327
```

Comment:
Available for fixing?

Reply:
Yes and no. It's really a hassle to not depend on any global node/react-native/browser types. We want types for `fetch` and `WebSocket`. We can either do that by depending on types for specific platform, or we should add them ourselves...

Reply:
Hmmm.. can we rely on global Node types, and then specify/override those in the respective environments? (my knowledge is a little light here so)

Reply:
I would rather avoid writing our own types, but I understand it's a tricky problem 🙁